devicetypes/sky-nie/min-smart-plug-dimmer.src/min-smart-plug-dimmer.groovy

```diff
@@ -0,0 +1,672 @@
+                attributeState "temperature", label: '${currentValue}°',
+                        backgroundColors:[
+                                [value: 31, color: "#153591"],
+                                [value: 44, color: "#1e9cbb"],
+                                [value: 59, color: "#90d2a7"],
+                                [value: 74, color: "#44b621"],
+                                [value: 84, color: "#f1d801"],
+                                [value: 95, color: "#d04e00"],
+                                [value: 96, color: "#bc2323"]
+                        ]
+            }
+        }
+
+		standardTile("refresh", "device.refresh", width: 2, height: 2) {
+			state "refresh", label:'Refresh', action: "refresh"
+		}
+		valueTile("syncStatus", "device.syncStatus", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "syncStatus", label:'${currentValue}'
+		}
+		standardTile("sync", "device.configure", width: 2, height: 2) {
+			state "default", label: 'Sync', action: "configure"
+		}
+		valueTile("firmwareVersion", "device.firmwareVersion", decoration:"flat", width:3, height: 1) {
+			state "firmwareVersion", label:'Firmware ${currentValue}'
+		}
+		valueTile("icon", "device.icon", inactiveLabel: false, decoration: "flat", width: 4, height: 1) {
+			state "default", label: '', icon: "https://inovelli.com/wp-content/uploads/Device-Handler/Inovelli-Device-Handler-Logo.png"
+		}
+		main "switch", "temperature"
+		details(["switch", "temperature", "refresh", "syncStatus", "sync", "firmwareVersion"])
+	}
+
+    simulator { }
+
+    preferences {
+        configParams.each {
+            if (it.name) {
+                if (it.range) {
+                    getNumberInput(it)
+                }
+                else {
+                    getOptionsInput(it)
+                }
+            }
+        }
+    }
+}
+
+private getOptionsInput(param) {
+    input "configParam${param.num}", "enum",
+            title: "${param.name}:",
+            required: false,
+            defaultValue: "${param.value}",
+            options: param.options
+}
+
+private getNumberInput(param) {
+    input "configParam${param.num}", "number",
+            title: "${param.name}:",
+            required: false,
+            defaultValue: "${param.value}",
+            range: param.range
+}
+
+def installed() {
+    logDebug "installed()..."
+    state.refreshConfig = true
+}
+
+def updated() {
+    if (!isDuplicateCommand(state.lastUpdated, 5000)) {
+        state.lastUpdated = new Date().time
+
+        logDebug "updated()..."
+
+        initialize()
+
+        runIn(5, executeConfigureCmds, [overwrite: true])
+    }
+
+    return []
+}
+
+private initialize() {
+    def checkInterval = ((60 * 60 * 3) + (5 * 60))
+
+    def checkIntervalEvt = [name: "checkInterval", value: checkInterval, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"]]
```

Comment:
In https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/69768/files#diff-ede2fdd21911fde9f7d3dd84c6f9f40dc678f9b3f53f81350d4f6a9c30230c5fR234 you wrote that this is _sleepy_ device, but here you've marked it as `offlinePingable`, which is definitely not true for sleepy devices. Aside from the fact that this option is reserved only for crucial devices (like home security), Dimmer Plug device doesn't seem to be a sleepy device.

So, to summarize, please remove `offlinePingable: "1"`, and...

Reply:
OK, I removed it.